api/MediaStreamTrack.json

```diff
@@ -1041,10 +1041,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
Ghost feature?

Reply:
I think this was in Chrome at some point, but that's not reflected in the data.